ast/option_editor.go

```diff
@@ -0,0 +1,83 @@
+package ast
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"time"
+)
+
+func NewOptionEditor(keyMap map[string]interface{}) *OptionEditor {
```

Comment:
Can you use `values.Value` here instead of `interface{}`?

Reply:
If I import `values` I get an import cycle: ast -> values -> ast.

Is there any workaround? Maybe externalize types in another module?